_includes/sqlstyle.guide.jp.md

```diff
@@ -1,3 +1,1196 @@
-# SQL style guide - Japanese translation
+{::options parse_block_html="true" /}
+<div lang="ja">
+# SQLスタイルガイド（日本語訳）
 
-...content goes here...
\ No newline at end of file
+**日本語訳について**[^japanese]
+
+[^japanese]: [Simon Holywell][simon]氏による[SQL Style Guide][self]の日本語訳です。日本語訳についての責は翻訳者である久利史之が負います。日本語訳は誤訳や[原文の最新版][self]に追随していない恐れがあります。誤訳や改善点があれば、[日本語訳レポジトリ][repo-jp]にPull RequestまたはIssueを登録するか、[twitter][twitter-jp]にてお知らせください。<br>この翻訳は[クリエイティブ・コモンズ 表示-継承4.0国際ライセンス][licence]下に提供されています。
```

Comment:
I don't think that this should be a footnote and instead I think it would be better to just have it inline at the top of the document.

So something like:

```markdown
## 日本語訳について

[Simon Holywell][simon]氏による[SQL Style Guide][self]の日本語訳です。日本語訳についての責は翻訳者である久利史之が負います。日本語訳は誤訳や[原文の最新版][self]に追随していない恐れがあります。誤訳や改善点があれば、[日本語訳レポジトリ][repo-jp]にPull RequestまたはIssueを登録するか、[twitter][twitter-jp]にてお知らせください。<br>この翻訳は[クリエイティブ・コモンズ 表示-継承4.0国際ライセンス][licence]下に提供されています。
```

Again though I think it is better if issues and pull requests are lodged against this repository and not your fork so that they are easier to track. I can always assign them to you and/or ping you on twitter.

You shouldn't need to mention the licence again here as it is stated at the base of the document.

I'd be tempted to format this section something like this:

```markdown
## 日本語訳について

Translator: [Nkuri](https://github.com/nkurigit) - [twitter](https://twitter.com/nkuritw)

There is a danger that the Japanese translation does not follow the latest version of the original text or there might be a mistranslation. If there is some mistranslation or improvement points, please register a pull request or Issue.
```

I cannot read or write Japanese (I am using Google Translate to read your submission) so you'd need to translate my English in the above suggestion of course.